app/models/Sprint.php

```diff
@@ -14,6 +15,11 @@ public function project()
 		return $this->belongsTo('Project');
 	}
 
+	public function sprintSnapshots()
```

Comment:
It's a laravel thing which lets me access the SprintSnapshots of a Sprint through a dynamic property http://laravel.com/docs/4.2/eloquent#dynamic-properties